frontend/public/components/monitoring/receiver-forms/email-receiver-form.tsx

```diff
@@ -249,19 +245,55 @@ export const Form: React.FC<FormProps> = ({ globals, formValues, dispatchFormCha
           </label>
         </div>
       </div>
+      <div className="form-group">
+        <ExpandCollapse
+          textCollapsed="Show advanced configuration"
+          textExpanded="Hide advanced configuration"
+        >
+          <div className="co-form-subsection">
+            <div className="form-group">
+              <SendResolvedAlertsCheckbox
+                formField="email_send_resolved"
+                formValues={formValues}
+                dispatchFormChange={dispatchFormChange}
+              />
+            </div>
+            <div className="form-group">
+              <label className="control-label co-required" htmlFor="emailHtml">
+                Body of email notifications (HTML)
```

Comment:
Use consistent case for all labels

```suggestion
                Body of Email Notifications (HTML)
```